src/handlers/default.ts

```diff
@@ -98,7 +98,7 @@ export class InngestCommHandler {
   private readonly inngestRegisterUrl: URL;
 
   protected readonly frameworkName: string = "default";
-  protected readonly signingKey: string;
+  protected readonly signingKey: string | undefined;
```

Comment:
Is it valid to not provide a signing key? Do we not always need one?

Reply:
we do need one, though if you use `proces.env.INNGEST_SIGNING_KEY` or similar env variables it's still possible to be undefined, so we need to denote this and handle appropriately.